From a549987698d105fab2b2bdddeed49b586f4d799d Mon Sep 17 00:00:00 2001
From: Fabian Vogt <fvogt@suse.de>
Date: Fri, 4 Nov 2022 15:29:59 +0100
Subject: [PATCH] Fix tools/Makefile with GNU make 4.4

Before GNU make 4.4, "export" variables were not actually exported to $(shell)
commands, so the "export GO111MODULE=off" line had no effect. With
make >= 4.4, this variable is now set in the environment of the "go build"
command, which breaks it. Just remove the wrong variable assignment to have
the same behavour as previously.

Signed-off-by: Fabian Vogt <fvogt@suse.de>
---
 tools/Makefile | 2 --
 1 file changed, 2 deletions(-)

diff --git a/tools/Makefile b/tools/Makefile
index 25783aef..0211016a 100644
--- a/tools/Makefile
+++ b/tools/Makefile
@@ -1,5 +1,3 @@
-export GO111MODULE=off
-
 GO := go
 
 BUILDDIR := build
